models/Item.js

```diff
@@ -72,17 +72,18 @@ Item.checkStatus = (serial_no, status) => {
 
 // Filtering
 Item.filter = data => {
-	const { is_broken, status } = data;
+	const { is_broken, status, type } = data;
 	let brokenFilter = is_broken
 		? is_broken === "true"
 			? `"item"."is_broken"`
 			: `NOT "item"."is_broken"`
 		: null;
 	let statusFilter = status ? `"item"."status" = ${status}` : null;
+	let typeFilter = type ? `"model"."is_product_type_name" = ${type}` : null;
```

Comment:
Should be 
```'${type}'``` instead of ```${type}```
Quotes are required for strings in SQL. 
By the way, ```${status}``` should be ```'${status}'``` as well, but I forgot. Sorry!